lib/js/app/queryCreator/components/Analysis/components/ListItem.tsx

```diff
@@ -26,12 +20,8 @@ type Props = {
   onClick: (e: React.MouseEvent<HTMLLIElement>, analysis: Analysis) => void;
   /** React children nodes */
   children: React.ReactNode;
-};
-
-const hintMotion = {
-  initial: { opacity: 0, right: -24 },
-  animate: { opacity: 1, right: -15 },
-  exit: { opacity: 0 },
+  /** Show hint handler */
+  showHint: (value: boolean, topPos?: number, bottomPos?: number) => void;
```

Comment:
Based on the `type` the topPos and bottomPos are not always exist - however this arguments are always passed in `showHint` function invocation. Please, adjust function type.

Reply:
`topPos` and `bottomPos` are not passed in `showHint`, `onMouseLeave`